tests/unit_tests/test_engine.py

```
@@ -79,7 +84,7 @@ def get_content_text(response, content_type):
 
 
 @pytest.mark.parametrize("use_www", [True, False])
-def test_www_redirects(use_www):
+def test_www_redirects(use_www: bool):
```

Ruff (0.14.10)

Boolean-typed positional argument in function definition

(FBT001)